code/controllers/master.dm

```diff
@@ -55,13 +68,19 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 	/// The type of the last subsystem to be fire()'d.
 	var/last_type_processed
 
-	var/datum/controller/subsystem/queue_head //!Start of queue linked list
-	var/datum/controller/subsystem/queue_tail //!End of queue linked list (used for appending to the list)
-	var/queue_priority_count = 0 //Running total so that we don't have to loop thru the queue each run to split up the tick
-	var/queue_priority_count_bg = 0 //Same, but for background subsystems
-	var/map_loading = FALSE //!Are we loading in a new map?
-
-	var/current_runlevel //!for scheduling different subsystems for different stages of the round
+	///!Start of queue linked list
+	var/datum/controller/subsystem/queue_head
+	///!End of queue linked list (used for appending to the list)
+	var/datum/controller/subsystem/queue_tail
+	///Running total of all queued subsystem priorities so that we don't have to loop thru the queue each run to split up the tick
+	var/queue_priority_count = 0
+	///running total of queued subsystem priority but for background subsystems
+	var/queue_priority_count_bg = 0
+	///!Are we loading in a new map?
+	var/map_loading = FALSE
+
+	///!for scheduling different subsystems for different stages of the round
```

Comment:
if you're gonna move these to before the thing, you have to remove the `!` from the comment as that tells codedocs to attach the comment to the *previous* thing, not the next thing.

Reply:
ah right

Reply:
fixed